.github/workflows/ci_release.yml

```diff
@@ -147,11 +147,11 @@ jobs:
       DOCSTRING: false
     secrets: inherit
 
-  pydpf-post_251:
-    name: "PyDPF-Post with 251"
+  pydpf-post_252:
```

Comment:
@guptadivya1501 We will also need retrocompatibility tests with 251, just like for 242 below